### ISSUE TITLE
<code> in data file translations doesn't work with semicolons

### ISSUE THREAD

**Issue by krzys-h** [colobot/colobot#810]
https://github.com/colobot/colobot-data/blob/master/i18n-tools/scripts/translate_help.py#L165

This regex is wrong, it doesn't work if there is any semicolon in the code block (and that happens quite often :stuck_out_tongue:). See for example:
https://github.com/colobot/colobot-data/blob/master/help/cbot/po/pl.po#L5925-L5935
```
msgid "Instruction <code>buildingenabled</code>"
msgid "The instruction <c/>buildingenabled();<n/> lets you know if a building of the given <a cbot|category>category</a> can be built in the level."
```

I guess @OdyX made that mistake when writing original translation code and @piotrdz just copied the regex as it was, because otherwise we would have probably lost some translations in the process.

This issue is extremly low priority, just noting something that needs to be fixed at some point.

**Comment by MrSimbax** [comment]
I think this regex should be just `\\c;(.*?)\\n;`, as you might want to put the escape sequences like `\n` inside too.

**Comment by krzys-h** [comment]
Yeah, the only reason I didn't change it immediately is that you also need to update all translation files too before applying this change.
